README.md

```diff
@@ -35,50 +38,61 @@ docker compose up
 psql postgres://postgres:hydra@127.0.0.1:5432
 ```
 
-### Or
+### Use Hydra Cloud
 
-Managed in the [cloud](https://hydra.so/).
+Hydra Cloud is the fastest and most reliable way to run Hydra. It is a cloud-based data warehouse that allows you to consolidate data from various sources into a single, unified system. It provides a user-friendly interface for automated data ingestion and transformation.
 
-## 📄 Documentation
+Hydra Cloud provides a scalable and secure cloud environment where automatic backups, resource scaling, high availability, point-in-time recovery, and more is available instantly with new databases.
 
-You can find our documentation [here](https://docs.hydra.so/getting-started/setup-guide).
+Sign up for [Hydra Cloud](https://dashboard.hydras.io/signup) and get a free, managed database.
 
-## 👩🏾‍🤝‍👨🏻 Community
+## 🎁 Features
+<br>🗃 hosted postgres database - [docs](https://docs.hydra.so/getting-started)
+<br>📎 columnar store with updates and deletes- [docs](https://docs.hydra.so/concepts/using-hydra-columnar)
+<br>🤹‍♀️ query parallelization
+<br>🐎 vectorized execution of WHERE clauses
+<br>📊 external tables - [docs](https://docs.hydra.so/concepts/using-hydra-external-tables)
 
-- [Discord chat](https://discord.com/invite/zKpVxbXnNY) for quick questions
-- [GitHub Discussions](https://github.com/hydradatabase/hydra/discussions) for longer topics
-- [GitHub Issues](https://github.com/hydradatabase/hydra/issues) for bugs and missing features
-- [@hydradatabase](https://twitter.com/hydradatabase) on Twitter
+## 🤝 Community and Status
+[DEVELOPERS.md](https://github.com/hydradatabase/hydra/blob/main/DEVELOPERS.md) for contributing and building the image.
+<br>[Discord](https://discord.com/invite/zKpVxbXnNY) discussion with the Community and Hydra team
+<br>[GitHub Discussions](https://github.com/hydradatabase/hydra/discussions) for longer topics
+<br>[GitHub Issues](https://github.com/hydradatabase/hydra/issues) for bugs and missing features
+<br>[Blog](https://blog.hydra.so/) for latest announcements, tutorials, product updates
+<br>[@hydradatabase](https://twitter.com/hydradatabase) for the tweets, memes, and social posts
+<br>[Docs](https://docs.hydra.so/) for Hydra features and warehouse ops
 
-## ✅ Status
-
-- [x] Early Access: Closed, private testing
-- [ ] Open Alpha: Open for everyone
-- [ ] Open Beta: Hydra can handle most non-enterprise use
-- [ ] Production: Enterprise ready
-
-We are currently in Early Access. Watch [releases](https://github.com/hydradatabase/hydra/releases) of this repo to get notified of updates.
+<br>
 
 ![follow the repo](.images/follow.gif)
```

Comment:
```suggestion
```

this image now feels a bit random / out of place. I don't think it's needed.

Reply:
Let's leave it for now- likely to remove after adding hydra architecture diagram